metadata.json

```diff
@@ -21,8 +21,8 @@
       "version_requirement": ">= 1.1.3 < 2.0.0"
     },
     {
-      "name": "nanliu/staging",
-      "version_requirement": ">= 1.0.0 < 2.0.0"
+      "name": "puppet-community/archive",
```

Comment:
Yes, you are right. That changed from the last time I edited it. All puppet-community forge modules were renamed.